<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="editor"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mnuMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actOpenFile"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuAnalyze"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockTC">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32000</width>
     <height>80</height>
    </size>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="parse_and_typecheck" name="dockWidgetParseAndTypeCheck">
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockREW">
   <property name="minimumSize">
    <size>
     <width>78</width>
     <height>42</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="rewriter" name="dockWidgetRewriter"/>
  </widget>
  <widget class="QDockWidget" name="dockSOL">
   <property name="minimumSize">
    <size>
     <width>78</width>
     <height>200</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="solver" name="dockWidgetSolver"/>
  </widget>
  <action name="actOpenFile">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>rewriter</class>
   <extends>QWidget</extends>
   <header>mcrl2xi_qt_rewriter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>solver</class>
   <extends>QWidget</extends>
   <header>mcrl2xi_qt_solver.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>parse_and_typecheck</class>
   <extends>QWidget</extends>
   <header>mcrl2xi_qt_parse_and_typecheck.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>onResetClicked()</slot>
  <slot>onTimer()</slot>
  <slot>onWidgetResized(float,float)</slot>
 </slots>
</ui>
